packages/react-components/src/theming/icons.tsx

```diff
@@ -119,6 +119,36 @@ const DomainPermissionCamera20Filled = (): JSX.Element => {
   );
 };
 
+/* @conditional-compile-remove(call-readiness) */
+const DomainPermissionsMicDenied20Filled = (): JSX.Element => {
+  const theme = useTheme();
+  return (
+    <Stack
+      horizontalAlign={'center'}
+      styles={{ root: { borderRadius: '100%', background: theme.palette.themeLighterAlt, padding: '2rem' } }}
```

Comment:
nit: this particular style is used 4 times. Maybe a good idea to create a common variable?

Reply:
Great callout, made a `icons.styles.ts` file and move common styles to there